logic/resolvable/Concludable.java

```diff
@@ -797,7 +777,7 @@ public Set<ValueConstraint<?>> values() {
         }
 
         @Override
-        FunctionalIterator<Unifier> unify(Rule.Conclusion conclusion, ConceptManager conceptMgr) {
+        public FunctionalIterator<Unifier> unify(Rule.Conclusion conclusion, ConceptManager conceptMgr) {
             if (conclusion.isValue()) return unify(conclusion.asValue());
```

Comment:
Public so LogicManager can access it. We should eventually move Concludable to the logic package?